packages/@glimmer/syntax/index.ts

```diff
@@ -32,6 +32,8 @@ export type { NodeVisitor } from './lib/traversal/visitor';
 export { default as Walker } from './lib/traversal/walker';
 export * as ASTv1 from './lib/v1/api';
 export { default as builders } from './lib/v1/public-builders';
+export { default as visitorKeys } from './lib/v1/visitor-keys';
+export { voidMap, voidTagNames } from './lib/generation/printer';
```

Comment:
If it's not needed for other users, we can remove `voidMap`. `voidTagNames` is enough for Prettier.

Reply:
:rogerroger: